com.unity.render-pipelines.high-definition/Editor/Material/LayeredLit/LayeredLitGUI.cs

```diff
@@ -22,8 +22,6 @@ internal struct SerializeableGUIDs
     class LayeredLitGUI : HDShaderGUI
     {
         const LitSurfaceInputsUIBlock.Features commonLitSurfaceInputsFeatures = LitSurfaceInputsUIBlock.Features.LayerOptions;
-        // Layered lit shaders don't support emission (realtime or baked)
```

Comment:
@sebastienlagarde  is that an out of date comment ? because emission field is currently displayed

Reply:
yes